tests/installation/setup_online_repos.pm

```diff
@@ -55,7 +61,10 @@ sub run() {
 
     send_key $cmd{next};    # Next
 
-    assert_screen "desktop-selection", 200;    # Make sure repos setup is finished and went to next step already
+    if (get_var("WITH_MAIN_REPO")) {
```

Comment:
As soon as the "need for a variable naming convention" is tracked somewhere else than coolo's brain I would merge it